### ISSUE TITLE
Regions on Sandbox

### ISSUE THREAD

**Issue by esfomeado** [dnsimple/dnsimple-java#78]
Hello,

I'm trying to create a zone record on my sandbox environment but I get an error:
"Regions are not allowed for your plan."

```
        ZoneRecordOptions zoneRecordOptions = ZoneRecordOptions.of(subdomain, "A", ip);
        dnsimpleClient.zones.createZoneRecord(dnsimpleConfig.getAccountId(),
                dnsimpleConfig.getMainDomain(), zoneRecordOptions);
```

So it seems by default the API uses the region **global** and there is no way at least of my knowledge to set to null the region.

```
    public static ZoneRecordOptions of(String name, String type, String content) {
        return new ZoneRecordOptions(name, type, content, null, null, singletonList("global"));
    }
```

Thanks

**Comment by ggalmazor** [comment]
Thanks for creating the issue, @esfomeado, and apologies for the delay in this answer.

`ZoneRecordOptions.of(String, String, String)` will produce a record configured to be distributed in all DNSimple's regions. You don't need to set the `regions` to `null` or an empty list for that.

If you try the same API operation in CURL and omit the `regions` argument, you will see how our system creates it in the `global` region for you even though you don't set one in your request.

In other words, `global` is the default value for `regions` when creating a zone record.

However, if you want to create the record in a set of regions, you can use `ZoneRecordOptions.regions` for that.

Does that solve your issue?

**Comment by ggalmazor** [comment]
Regarding the `regions are not allowed for your plan` message you're getting: did you subscribe to a plan in your sandbox account? You can use `4242 4242 4242 4242` as a credit card for that

**Comment by esfomeado** [comment]
Yes that solves the issue.

Thanks for the help.
